app/controllers/annotations_controller.rb

```diff
@@ -81,4 +81,15 @@ def show
     def exception_test
         raise "This is a test of the exception handler. If you received this email, it is working properly."
     end
+
+    private
+
+    def generate_csv(loadOptions, token)
+      data =  ApiRequester.search(loadOptions, token, to_csv: true) 
+      if data
+        send_data data
+      else
+        redirect_to :back, alert: "There are not any annotations to generate a csv file"
```

Comment:
I think we want this to say: "There are no annotations on this document"

Reply:
Or, as noted in the other PR: "There are no annotations to be downloaded."

Reply:
I think this was fixed in another PR -- this one is good to go. Merge, deploy, delete branch. Thanks.